packages/react-components/src/components/chart/chartOptions/useChartConfiguration.ts

```diff
@@ -0,0 +1,183 @@
+import {
```

Comment:
can we add some tests ? given this only deals with data

Reply:
This is an aggregation of hooks / converters we already had created and tested so I didn't think it was necessary. useSeriesAndYAxis is the most complicated part of this hook and we already test that.

Reply:
Fair